### PR TITLE
Remove Diagonal dpi from SDL_GetDisplayPhysicalDPI()

### PR DESCRIPTION
**Pull request by 1bsyl**
Remove diagonal DPI from public API and from internal backends

https://github.com/libsdl-org/SDL/issues/7147

### CODE REVIEW

**Comment by sezero** [comment]
What to do in sdl2-compat? Copy over `SDL_ComputeDiagonalDPI` ?

**Comment by 1bsyl** [comment]
I am looking into it !

**Comment by slouken** [comment]
Because of the lack of reliability, I'm considering removing this API entirely. Thoughts?

**Comment by slouken** [comment]
If we do keep it, I think we only need one DPI value, not separate values for horizontal and vertical.

File: src/video/SDL_video.c [hunk 1]
```diff
@@ -759,23 +759,39 @@ int SDL_GetDisplayUsableBounds(int displayIndex, SDL_Rect *rect)
...
+    }
+
+    if (vdpi) {
+       *vdpi = v;
+    }
+
+    return 0;
+
+error:
+    return SDL_SetError("Couldn't get DPI");;
```
Comment (slouken) [inline src/video/SDL_video.c:888]: Two semicolons...

**Comment by 1bsyl** [comment]
thinking again,
maybe the internal sys api should have the 3 parameters:
hdpi vdpi diagdpi.
so that we can compute diag from hdpi / vdpi, or the inverse.

**Pull request closed without merging**